robottelo/ui/rhai.py

```diff
@@ -12,3 +18,50 @@ def view_registered_systems(self):
             locators['insights.registered_systems']
         ).text
         return result
+
+    def register_client_to_rhai(self, activation_key, org):
+        with VirtualMachine(distro='rhel67') as vm:
+            # Download and Install ketello-ca rpm
+            vm.install_katello_cert()
+            vm.register_contenthost(activation_key, org)
+
+            # Red Hat Access Insights requires RHEL 6/7 repo and it not
+            # possible to sync the repo during the tests,
+            # adding a file in /etc/yum.repos.d/rhel6/7.repo
+
+            rhel6_repo = conf.properties.get('insights.rhel6_repo')
```

Comment:
Isn't it simpler to just do a dict lookup and let the lookup itself raise a `KeyError`?

```python
>>> from robottelo.common import conf
>>> conf.properties.get('foreman.admin.username')
'admin'
>>> conf.properties['foreman.admin.username']
'admin'
>>> conf.properties.get('foreman.admin.usernamee')
>>> conf.properties['foreman.admin.usernamee']
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'foreman.admin.usernamee'
```